packages/namekit-react/src/components/Tooltip.tsx

```diff
@@ -12,19 +12,21 @@ type Props = {
 export const DEFAULT_MAX_TOOLTIP_WIDTH = 400;
 
 export const TooltipPlacement = {
-  TOP: "top",
-  RIGHT: "right",
-  BOTTOM: "bottom",
-  LEFT: "left",
+  Top: "top",
+  Right: "right",
+  Bottom: "bottom",
+  Left: "left",
 } as const;
 
 export type TooltipPlacement =
   (typeof TooltipPlacement)[keyof typeof TooltipPlacement];
 
+export const DEFAULT_TOOLTIP_PLACEMENT = TooltipPlacement.Top;
+
 export function Tooltip({
   trigger,
   children,
-  placement = TooltipPlacement.TOP,
+  placement = DEFAULT_TOOLTIP_PLACEMENT,
```

Comment:
Defining a named constant is a good strategy for default params. It helps to make code and docs easier to automatically keep updated as things change.